test/unit/app/helpers/admin/republishing_helper_test.rb

```diff
@@ -34,11 +34,17 @@ class Admin::RepublishingHelperTest < ActionView::TestCase
     assert_equal first_bulk_content_type, "All documents"
   end
 
-  test "#republishing_index_bulk_republishing_rows creates a link to the specific bulk republishing confirmation page" do
-    first_link = republishing_index_bulk_republishing_rows.first[1][:text]
-    expected_link = '<a id="all-documents" class="govuk-link" href="/government/admin/republishing/bulk/all-documents/confirm">Republish <span class="govuk-visually-hidden">all documents</span></a>'
+  # this can be removed when rebased on the by type branch
+  test "#republishing_index_bulk_republishing_rows creates a link to the new page for content types that require extra input" do
+    all_documents_by_organisation_row = republishing_index_bulk_republishing_rows.find do |row|
+      description_text = row[0][:text]
```

Comment:
[Another hotly debated one](https://github.com/alphagov/whitehall/pull/9122#discussion_r1642821077)

Reply:
I've added a separate commit to tidy things up in line with the approach you took in the other PR: https://github.com/alphagov/whitehall/pull/9150/commits/ed53a1770220a331565c39bc794c686a655d8d0f